Sources/RInAppMessaging/Router.swift

```diff
@@ -43,7 +50,7 @@ internal protocol RouterType: ErrorReportable {
 /// Handles all the displaying logic of the SDK.
 internal class Router: RouterType, ViewListenerObserver {
 
-    private enum UIConstants {
```

Comment:
@imaginaris What is the reason of removing `private`?

Reply:
Those constants are accessed by `TooltipViewModifier`. Actually I should move this enum